FluentUI.Demo/src/androidTest/java/com/microsoft/fluentuidemo/demos/V2PeoplePickerUITest.kt

```diff
@@ -107,9 +110,11 @@ class V2PeoplePickerUITest {
             )
         }
 
-        val peoplePickerChip = composeTestRule.onNodeWithText("firstName", true, useUnmergedTree = true)
+        val peoplePickerChip =
```

Comment:
space formatting

Reply:
Formatting file change. not added intentionally.